postgresql/hooks/run

```diff
@@ -1,6 +1,5 @@
 #!{{pkgPathFor "core/bash"}}/bin/bash
-
-set -e
```

Comment:
Any reason to drop the `set -e`? This should cause the `run` hook to abort if anything being set up returns with a non-zero exit code.

Reply:
nope error because of frankensteining the plan. Will correct.